example/example.config.js

```diff
@@ -63,11 +63,29 @@ const modules = [
     isResponsive: true
   }),
   colors({
+    nestedRules: {
+      ".dark {}": {
+        whitelist: ["c"],
+        isResponsive: true,
+        colorValues: {
+          black: "#ffffff",
+          white: "#1a1a1a"
+        }
+      },
+      "@media foo": {
```

Comment:
@medias are an exception and are handled separately without templating as the name of the rule remain unaffected, but it will get wrapped in a atRule